apps/mdn/mdn-aws/docs/mdn-dd-redis.md

```diff
@@ -0,0 +1,98 @@
+### How does this work?
```

Comment:
Can we add a "why" paragraph? Something like:

MDN uses Redis as the message transport for Celery, to store tasks that are run in the background. Each Celery queue is stored as a list in Redis. By tracking the length of these lists in Datadog, we can identify trends over time, and alert staff when queued tasks are overwhelming the processing capacity.